Sources/PackagePlugin/PackageModel.swift

```diff
@@ -206,19 +236,24 @@ public enum TargetDependency {
     case product(Product)
 }
 
-/// Represents a target consisting of a source code module, containing either
-/// Swift or source files in one of the C-based languages.
+/// A target consisting of a source code module.
+///
+/// The module contains either Swift or source files in one of
+/// the C-based languages.
 public protocol SourceModuleTarget: Target {
-    /// The name of the module produced by the target (derived from the target
-    /// name, though future SwiftPM versions may allow this to be customized).
+    /// The name of the module produced by the target.
+    ///
+    /// Derived from the target name, though future SwiftPM versions may
```

Comment:
Since these are API docs I'm unsure as to whether we want to keep the `SwiftPM` naming or if we prefer `Swift package manager` here.